### ISSUE TITLE
New Functions management experience impacts multiple labs

### ISSUE THREAD

**Issue by MikeBazMSFT** [MicrosoftLearning/AZ-204-DevelopingSolutionsforMicrosoftAzure#45]
This impacts many steps in many labs - the new Functions management UI, which is basically the App Service management UI with some tweaks, was made the default after being in preview for a while.  This means that most instructions in the AK and in some cases the non-AK around how to get to administrative management of Functions are wrong, in some cases very much so.  A short-term fix is to tell the students to click the "Having issues" link at the top of the Overview of the new experience, but the proper long term fix is to revisit every use of Functions in every lab and fix the instructions.  (This is a big fix.)

**Comment by seesharprun** [comment]
Absolutely, we are underwater making fixes now. I’ll give an update in a couple of days.

**Comment by MikeBazMSFT** [comment]
Just adding a reference to #48 since that appears to be the meta-issue that relates to this one.
